pkgs/development/python-modules/aerosandbox/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  numpy,
+  scipy,
+  pandas,
+  matplotlib,
+  seaborn,
+  tqdm,
+  sortedcontainers,
+  dill,
+  casadi,
+}:
+
+buildPythonPackage rec {
+  pname = "aerosandbox";
+  version = "4.2.5";
+  format = "wheel";
+
+  src = fetchPypi {
+    pname = "AeroSandbox";
+    inherit version format;
+
+    python = "py3";
+    dist = "py3";
+    hash = "sha256-9WxeXmcOZpKpNn8r6REudQlDRXavym52tvKajYJIfXA=";
+  };
+
+  build-system = [ setuptools ];
```

Comment:
```suggestion
```
We don't need `build-system` when we use `format = "wheel"` (which means, `pyproject = false`).
This is because `pypaBuildHook` is not used in this case.
https://github.com/NixOS/nixpkgs/blob/cb8e0012488c0947bcfbcef402faf65259a53011/pkgs/development/interpreters/python/mk-python-derivation.nix#L263-L271